test/api/test_history_contents.py

```diff
@@ -1,11 +1,10 @@
-from base import api
 import json
 
-from .helpers import TestsDatasets
-from .helpers import LibraryPopulator
-from .helpers import DatasetCollectionPopulator
-from requests import delete
-from requests import put
+from requests import delete, put
+
+from base import api
+
+from .helpers import DatasetCollectionPopulator, LibraryPopulator, TestsDatasets
```

Comment:
So the more common source layout for Python packages is to have packages at the top level of the source, although a `src/` or `lib/` directory is not uncommon. I think going forward, though, we'd probably move the contents of `lib/` up one level. With that in mind, it's probably not necessary to put it in `lib/`.

But this is just my opinion.